src/libs/Navigation/AppNavigator/MainDrawerNavigator.js

```diff
@@ -85,3 +90,4 @@ export default compose(
         },
     }),
 )(MainDrawerNavigator);
+export {getInitialReportScreenParams};
```

Comment:
Old PR now, but any idea why this export was added? Doesn't seem to be used anywhere AFAICT

Reply:
Maybe we were using it somewhere at some point. I don't really know tbh.

Reply:
Originally it was part of `reportUtils` and imported to `MainDrawerNavigator`: https://github.com/Expensify/App/commit/bc63c492e068d8b2a1c3361579e13d5894f3ca8b#diff-82f2652821babb48b600a87486fa1191db7b3cc300845573a5757676e13cfc26L44

I decided it's tied to the MainDrawerNavigator (see `_.once` decorator and description) and moved it for better context
I guess I used automatic refactoring, which preserves exporting the file after it's moved